front-vite-react/front-vite-react/src/Pages/Logout.jsx

```diff
@@ -19,7 +19,7 @@ const Logout = () => {
         fetch('http://127.0.0.1:8000/api/logout', {
```

Comment:
Dica: Coloque a URL da API como uma variável global. FIca mais fácil de usar e se precisar trocar, troca no projeto inteiro. Isso é muito útil quando for alternar entre ambiente Desenvolvimento e Produção.